x-pack/plugins/security/public/nav_control/nav_control.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { CoreStart } from 'src/core/public';
+
+import { AuthenticatedUser } from '../../common/model';
+import { SecurityNavControl } from './nav_control_component';
+
+export function registerSecurityNavControl(
+  core: Pick<CoreStart, 'chrome' | 'http' | 'i18n' | 'application'>,
+  user: Promise<AuthenticatedUser>
+) {
+  core.chrome.navControls.registerRight({
+    order: 2000,
+    mount: (el: HTMLElement) => {
+      const I18nContext = core.i18n.Context;
+
+      const props = {
+        user,
+        editProfileUrl: core.http.basePath.prepend('/app/kibana/#account'),
```

Comment:
I was going to also move the account management page as part of this PR, but it ended up turning into a larger project than I expected, so I'm keeping it in the LP for now to make this easier to review and merge.

Once we move it to the NP, we should decouple it from the `/app/kibana` route.